lib/http/apps/wallet.js

```diff
@@ -15,11 +15,21 @@ app.post(`/`, (req, res) => {
       to: wallet.address,
       value: ethers.utils.parseEther('1')
     }).then(_ => {
-      res.status(201).json({ address: wallet.address });
+      res.status(201).json({ 
+        address: wallet.address,
+        mnemonic: wallet.mnemonic
+      });
     }).catch(console.log);
   }).catch(e => {
     console.log(e);
-    res.sendStatus(500);
+    if(e.errno == -17) {
+      res.status(500).json({
```

Comment:
we should use a different HTTP status here.
5xx is if something is wrong on the server side
4xx is if the client sent a bad request - which is here the case. maybe [400 bad request](https://http.cat/400) or [409 conflict](https://http.cat/409)